docs/tmp.js.html

```diff
@@ -491,7 +498,7 @@ <h1 class="page-title">Source: tmp.js</h1>
  */
 function _generateTmpName(opts) {
 
-  const tmpDir = _getTmpDir();
+  const tmpDir = opts.tmpdir;
```

Comment:
I would simply pass the options to the `_getTmpDir` function and return the required directory from there.

Reply:
done